website/src/css/custom.css

```diff
@@ -107,3 +107,65 @@ svg.lightToggleIcon_node_modules-\@docusaurus-theme-classic-lib-next-theme-Color
 li.footer__item a svg {
   display: inline;
 }
+
+/* mo changes 20 oct 2022 */
+
+.alert {
+  max-width: 1500px;
+  margin: 2em auto;
+}
+
+html[data-theme='dark'] .navbar {
+  background-color: #1b1b1d !important;
+}
+
+.navbar,
+.title-font,
+.text-base {
+  font-family: 'Montserrat' !important;
+}
+
+.text-base {
+  font-weight: 300;
+  font-size: 1.3em !important;
+}
+
+.hero .title-font {
+  font-weight: 600;
+  font-size: 3em !important;
+  margin-bottom: 0.5em;
+}
```

Comment:
I think this should be adressed directly in the tailwind/html code

Reply:
I ended up dropping both - I had ended up overriding these anyway inline so these weren't really being used.

Reply:
let's remove them then